src/Zypper.cc

```diff
@@ -900,6 +924,12 @@ void print_main_help( Zypper & zypper )
     "\tzypper <subcommand> [--command-options] [arguments]\n"
   );
 
+  static std::string help_keys_commands = _("     Key Management:\n"
+    "\tkeys, lk\t\tList all keys.\n"
+    "\taddkey, as\t\tAdd a new key to trust.\n"
+    "\tremovekey, rs\tRemove a key from trust.\n"
```

Comment:
Short names `as/rs` are wrong (copied from asdd/remove service?). Must be `ak/rk`

Reply:
Done